apps/ui/src/components/Modal/Vote.vue

```diff
@@ -88,6 +88,16 @@ watch(
   { immediate: true }
 );
 
+watch(
+  () => props.proposal,
+  (toProposal, fromProposal) => {
+    if (props.open && fromProposal?.id !== toProposal.id) {
+      emit('close');
+    }
+  },
+  { immediate: true }
+);
+
```

Comment:
We still leave this in edit state on change of proposals. maybe we should reset this as well. 

<img width="354" alt="Untitled 2" src="https://github.com/user-attachments/assets/593d4f55-03e2-4228-88a0-7dd3eeddfe75">